### PR TITLE
add custom mutator function for running script after target gets executed

### PR DESCRIPTION
**Pull request by yangzao**
Hi @vanhauser-thc, I've modified my code as you mentioned in #1910. Do you think the code is fine or if I need to change anything? Thanks!

### CODE REVIEW

File: src/afl-fuzz-run.c [hunk 1]
```diff
@@ -60,6 +60,8 @@ fuzz_run_target(afl_state_t *afl, afl_forkserver_t *fsrv, u32 timeout) {
 
   fsrv_run_result_t res = afl_fsrv_run_target(fsrv, timeout, &afl->stop_soon);
 
+  post_run(afl);
```
Comment (vanhauser-thc) [inline src/afl-fuzz-run.c]: Please do not make a function, put the code here

**Comment by vanhauser-thc** [comment]
Looks good! Just one minor thing :)
Did you try both the python and custom lib api for your function?

**Comment by yangzao** [comment]
> Looks good! Just one minor thing :) Did you try both the python and custom lib api for your function?

Thanks! I've tested both of them and they worked fine.

**Comment by vanhauser-thc** [comment]
Looks good, thank you!

**Pull request merged**